Index.html

```diff
@@ -1,269 +1,176 @@
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+  <script src="https://kit.fontawesome.com/bee9655978.js" crossorigin="anonymous"></script>
+  <meta name="Portfolio" content="this is a portfolio by Mohammad Mehdi Pazouki.">
+
+  <title>Mohammad Mehdi Pazouki</title>
 </head>
-<body>
-    <header class="position__nav toolbar">
-        <div class="corner__geometry  mobile__header">
-            <img class="mobile__visible" src="./Contents/Resources/Pictures/image_geometry_1.svg" alt="">
-            <img class="mobile__invisible" src="./Contents/Resources/Pictures/image_geometry_1.svg" alt="">
-        </div>
-        <!-- only add the toolbar__open__rowx using javascript -->
-        <nav class="drop-down menu__button__align mobile__header">
-            <button id="hamburger">
-                <i class="fa fa-bars closed__menu"></i>
-                <img class="opened__menu cross" src="./Contents/Resources/Icons/cross.svg" alt="">
-            </button>
-        </nav>
-
-        <img class="mobile__photos mobile__invisible" src="./Contents/Resources/Icons/nav_dots.svg" alt="">
 
-        <ul class="mobile__invisible desktop__header ">
-            <li class="header__navication"><a class="nav__desktoplink" href="#headline">Hello</a></li>
-            <li class="header__navication"><a class="nav__desktoplink" href="#works">Works</a></li>
-            <li class="header__navication"><a class="nav__desktoplink" href="#about">About</a></li>
-            <li class="header__navication"><a class="nav__desktoplink" href="#contact">Contact</a></li>
+<body>
+  <header id="header">
+    <nav id="header_nav">
+      <i class="fa-solid fa-bars" id="hamburger-menu"></i>
+      <ul id="desktop-nav">
+        <li>
+          <a href="#intro">Hello</a>
+        </li>
+        <li>
+          <a href="#Projects">Portfolio</a>
+        </li>
+        <li>
+          <a href="#about">About</a>
+        </li>
+        <li>
+          <a href="#contact-form">contact</a>
+        </li>
+      </ul>
+      <i class="fa-solid fa-xmark" id="closeMenuBtn"></i>
+      <ul id="mobile-nav">
+        <li>
+          <a href="#intro">Hello</a>
+        </li>
+        <li>
+          <a href="#Projects">Portfolio</a>
+        </li>
+        <li>
+          <a href="#about">About</a>
+        </li>
+        <li>
+          <a href="#contact-form">contact</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <section id="intro">
+    <div>
+      <h1><b>Mohammad Mehdi Pazouki</b></h1>
+      <p>
+        Hello! I am a software developer! I can help you build a product,
+        feature or website. Take a look of my works. If you like what you see
+        and have a project you need coded, don't hesitate and contact me.
+      </p>
+      <button type="button" class="dark small">Start collaboration</button>
+      <div id="logo"></div>
+    </div>
+  </section>
+  <section id="Projects">
+    <h1>Projects</h1>
```

Comment:
- [ ] Kindly as requested by the previous reviewer consider making sure that you stick to the Figma design as much as possible mainly here am talking about your mobile version consider making sure that as you develop the desktop version make sure that the mobile version is not messed up  that will make your work more professional
- check on this part 
your design 👇 
<img width="427" alt="Screen Shot 2023-05-04 at 21 33 21" src="https://user-images.githubusercontent.com/60146030/236311607-1b14487c-1b65-4e8a-b2e3-7034d1af742c.png">



Figma design 👇 
<img width="465" alt="Screen Shot 2023-05-04 at 21 33 54" src="https://user-images.githubusercontent.com/60146030/236311786-400f4a41-0a2f-4ffa-a427-25e376a36d5b.png">


consider doing so on the desktop version as well

- your design 👇 
<img width="1344" alt="Screen Shot 2023-05-04 at 21 47 35" src="https://user-images.githubusercontent.com/60146030/236314940-481fe57c-7ba8-4712-9bc2-84cc45768aa4.png">

- Figma design 👇 

<img width="867" alt="Screen Shot 2023-05-04 at 21 49 15" src="https://user-images.githubusercontent.com/60146030/236315106-cbdf67f0-06f7-40a5-9c87-0f54eb9e28be.png">

Reply:
> * [x]  Kindly as requested by the previous reviewer consider making sure that you stick to the Figma design as much as possible mainly here am talking about your mobile version consider making sure that as you develop the desktop version make sure that the mobile version is not messed up  that will make your work more professional
> * check on this part
>   your design point_down
> 
> <img alt="Screen Shot 2023-05-04 at 21 33 21" width="427" src="https://user-images.githubusercontent.com/60146030/236311607-1b14487c-1b65-4e8a-b2e3-7034d1af742c.png">
> 
> Figma design point_down <img alt="Screen Shot 2023-05-04 at 21 33 54" width="465" src="https://user-images.githubusercontent.com/60146030/236311786-400f4a41-0a2f-4ffa-a427-25e376a36d5b.png">
> 
> Consider doing so on the desktop version as well
> 
> * your design point_down
> 
> <img alt="Screen Shot 2023-05-04 at 21 47 35" width="1344" src="https://user-images.githubusercontent.com/60146030/236314940-481fe57c-7ba8-4712-9bc2-84cc45768aa4.png">
> 
> * Figma design point_down
> 
> <img alt="Screen Shot 2023-05-04 at 21 49 15" width="867" src="https://user-images.githubusercontent.com/60146030/236315106-cbdf67f0-06f7-40a5-9c87-0f54eb9e28be.png">

- [ ] As the previous reviewers suggested, Kindly try to match your design to the Figma design. :+1: